code/drasil-gen/lib/Database/Drasil/ChunkDB.hs

```diff
@@ -107,5 +108,8 @@ cdb s t c u d ins gd tm ci lc r cits =
   $ insertAll tm
   $ insertAll ci
   $ insertAll cits
+  $ insert (inReq EmptyS) 
```

Comment:
This is always a serious code smell. This embeds a bad assumption too deeply, i.e. will make gamephysics harder still.